### PR TITLE
[MRG] remove scikit-learn and numpy dependencies for normalization

### PR DESCRIPTION
**Pull request by vc1492a**

### CODE REVIEW

**Review by adidier17 — APPROVED** [review]

**Review by adidier17 — APPROVED** [review]
Looks good. Thanks for removing the SkLearn dependency
